src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -55,121 +54,65 @@ import { SpacesDropdown } from "components/atoms/SpacesDropdown";
 import "./SpaceEditForm.scss";
 
 const HANDLED_ERRORS = [
-  "room.template",
-  "room.title",
-  "room.subtitle",
-  "room.about",
-  "room.url",
-  "room.image_url",
-  "venue.mapBackgroundImage",
-  "venue.iframeUrl",
-  "venue.zoomUrl",
-  "venue.auditoriumColumns",
-  "venue.auditoriumRows",
-  "venue.columns",
+  "name",
+  "title",
+  "subtitle",
+  "image_url",
+  "mapBackgroundImage",
+  "iframeUrl",
+  "zoomUrl",
+  "auditoriumColumns",
+  "auditoriumRows",
+  "columns",
```

Comment:
```suggestion
  "name",
  "subtitle",
  "mapBackgroundImage",
  "iframeUrl",
  "zoomUrl",
  "auditoriumColumns",
  "auditoriumRows",
  "columns",
```

title, image_url - are the leftovers from the portal data.